.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Add your VSCode folder to the repo's gitignore, or delete this file from the repo.